xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/AccountTransactionsIT.java

```diff
@@ -39,20 +39,20 @@
 import org.xrpl.xrpl4j.model.client.ledger.LedgerResult;
 import org.xrpl.xrpl4j.model.transactions.Address;
 import org.xrpl.xrpl4j.model.transactions.Hash256;
-import org.xrpl.xrpl4j.tests.environment.MainnetEnvironment;
+import org.xrpl.xrpl4j.tests.environment.TestnetEnvironment;
```

Comment:
@mukulljangid What instigated this change?

Unless there's a really good reason to prefer Testnet, I propose that we revert this change because if Testnet ever gets reset we'll need to go in and adjust this test (and I can't think of any reason _not_ to hit `Mainnet`.

WDYT?

Also, @nkramer44 WDYT?

Reply:
I didn't think of the reset possibility. Mainnet requests were usually pretty slow and led tests to fail.

Reply:
Interesting -- I would expect the move to clio in s1/s2 to make Mainnet requests much more responsive, FWIW.